equistore-torch/src/array.cpp

```diff
@@ -1,44 +1,95 @@
 #include <vector>
 #include <cstdint>
 
-#include <torch/torch.h>
+#include <torch/script.h>
 
 #include <equistore.hpp>
 
 #include "equistore/torch/array.hpp"
 
 using namespace equistore_torch;
 
+eqs_data_origin_t equistore_torch::TORCH_DATA_ORIGIN = 0;
```

Comment:
Can this get a bit more explanation? This is not clear to me how this constants decided?

Reply:
this is just initialized to 0 here (meaning "no origin"), and later allocated by the code in `EqsOriginRegistration`

Reply:
I'll add some more comment around this